lisa/util/constants.py

```diff
@@ -47,6 +47,7 @@
 # notifier
 NOTIFIER = "notifier"
 NOTIFIER_CONSOLE = "console"
+NOTIFIER_TEXT_RESULT = "text_result"
```

Comment:
We can move this to `type_name`. If a class needs this constant, they can call `type_name` method directly.

Reply:
@somil55 I noticed that the html notifier type_name handles this constant similarly to what you want me to do for my notifier. Why are we doing this differently for text_result and html instead of keeping these constants in constants.py like how it is for notifier and console notifier?

Reply:
I am not sure. My understanding was that class names are uniquely mentioned in `type_name` method. @squirrelsc do you know the answer to this?